cmd/internal/storage/influxdb/influxdb_test.go

```diff
@@ -226,7 +226,7 @@ func TestContainerStatsToPoints(t *testing.T) {
 	points := storage.containerStatsToPoints(cInfo, stats)
 	points = append(points, storage.memoryStatsToPoints(cInfo, stats)...)
 	points = append(points, storage.hugetlbStatsToPoints(cInfo, stats)...)
-	points = append(points, storage.perfStatsToPoints(cInfo, stats)...)
+	points = append(points, storage.Perf.PerfStatsToPoints(cInfo, stats)...)
```

Comment:
`PerfStatsToPoints` does not exist.
Have you run tests for InfluxDB? Have you seen [this](https://github.com/google/cadvisor/blob/2e9277ba558bb56e716fe84406c8f0c83cecc7bd/cmd/internal/storage/influxdb/influxdb_test.go#L16) instruction?